lib/rubocop/config_validator.rb

```diff
@@ -102,6 +102,17 @@ def check_target_ruby
     end
 
     def alert_about_unrecognized_cops(invalid_cop_names)
+      unknown_cops = list_unknown_cops(invalid_cop_names)
+
+      if ConfigLoader.ignore_unrecognized_cops
+        warn Rainbow('Ignoring unrecognized cops or departments by option.').yellow
```

Comment:
I'd use wording like "The following cops or departments are not recognized and will be ignored:".

Reply:
Thank you! I've updated the message.